exercises/alphametics/example.rb

```diff
@@ -0,0 +1,37 @@
+class Alphametics
+  VERSION = 1
+
+  def solve(puzzle)
+    letters = Hash.new(0)
+    puzzle = format_to_ruby(puzzle)
+    puzzle.scan(/[a-zA-Z]/) { |w| letters[w] += 1 }
+
+    possible_values(letters.keys) do |letters_values|
+      return letters_values if valid?(puzzle, letters_values)
+    end
+
+    nil
+  end
+
+  private
+
+  def format_to_ruby(puzzle)
+    puzzle.gsub(/\^/, '**') # format exponent: 2 ^ 3 to 2 ** 3
```

Comment:
Rubocop is suggesting `tr` instead of `gsub`, but I can't see how.

@kytrinyx @kotp any suggestion?